tests/unit/utils/txn_parser/test_invalid_txn.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from copy import deepcopy
+from unittest import TestCase
+
+import xrpl.utils
+from tests.unit.utils.txn_parser.transaction import TXN
+
+meta_not_included_txn = deepcopy(TXN)
+meta_not_included_txn.pop("meta")
+no_nodes_txn = deepcopy(TXN)
+no_nodes_txn["meta"].clear()
+
+
+class TestInvalidTxn(TestCase):
```

Comment:
Tests that are testing the same method should be in the same file. If you want, you can still have both classes in that file, instead of combining them into one class.